firebase-functions/src/main/java/com/google/firebase/functions/FirebaseFunctions.java

```diff
@@ -187,7 +193,7 @@ URL getURL(String function) {
    *
    * @param origin The origin of the local emulator, such as "http://10.0.2.2:5005".
    */
-  public void useFunctionsEmulator(String origin) {
+  public void useFunctionsEmulator(@NonNull String origin) {
     urlFormat = origin + "/%2$s/%1$s/%3$s";
```

Comment:
might make sense to add a precondition check on `origin`?

Reply:
Done.